Multisig/UI/WalletConnect/Client/WCPendingConfirmationViewController.swift

```diff
@@ -102,6 +98,23 @@ class WCPendingConfirmationViewController: UIViewController {
         }
     }
 
+    func send(completion: @escaping (String?) -> Void) {
```

Comment:
when do we call this method?

Reply:
please use the search

Reply:
Check the ReviewExecutionViewController file in this pr